amqp/connection.py

```diff
@@ -83,7 +83,13 @@ class Connection(AbstractChannel):
 
     Authentication can be controlled by passing one or more
     `amqp.sasl.SASL` instances as the `authentication` parameter, or
-    by using the userid and password parameters (for AMQPLAIN and PLAIN).
+    setting the `login_method` string to one of the supported methods:
+    'GSSAPI', 'EXTERNAL', 'AMQPLAIN', or 'PLAIN'.
+    Otherwise authentication will be performed using any supported method
+    preferred by the server. Userid and passwords apply to AMQPLAIN and
+    PLAIN authentication, whereas on GSSAPI only userid will be used as
+    client name. For EXTERNAL authentication both userid and password are
```

Comment:
Please adjust comment to "whereas on GSSAPI only userid will be used as **the** client name"